src/libmongoc/tests/unified/operation.c

```diff
@@ -303,9 +356,10 @@ operation_run_command (test_t *test,
    mongoc_read_concern_t *rc = NULL;
    bson_error_t op_error = {0};
    bson_t op_reply = BSON_INITIALIZER;
-   bson_t opts = BSON_INITIALIZER;
+   bson_t *opts = NULL;
 
    parser = bson_parser_new ();
+   bson_parser_allow_extra (parser, true);
```

Comment:
Unless this is needed, I suggest removing the call to `bson_parser_allow_extra`. `mongoc_client_session_append` can be called on `opts` directly without calling `bson_parser_get_extra`. Since this was validating that no extra keys were included in the arguments before, `bson_parser_get_extra` should be an empty document.

Reply:
Removed!